README.md

```diff
@@ -92,146 +93,52 @@ official compatibility chart below.
-
-#### Bindings examples
-
-```
-# Comma separated string. Exposes, 'color' and 'size'
-$ kubernetes-deploy my-app cluster1 --bindings=color=blue,size=large
-
-# JSON string. Exposes, 'color' and 'size'
-$ kubernetes-deploy my-app cluster1 --bindings='{"color":"blue","size":"large"}'
-
-# Load JSON file from ./config
-$ kubernetes-deploy my-app cluster1 --bindings='@config/production.json'
-
-# Load YAML file from ./config (.yaml or .yml supported)
-$ kubernetes-deploy my-app cluster1 --bindings='@config/production.yaml'
-```
-
-
-#### Using partials
-
-`kubernetes-deploy` supports composing templates from so called partials in order to reduce duplication in Kubernetes YAML files. Given a template directory `DIR`, partials are searched for in `DIR/partials`and in 'DIR/../partials', in that order. They can be embedded in other ERB templates using the helper method `partial`. For example, let's assume an application needs a number of different CronJob resources, one could place a template called `cron` in one of those directories and then use it in the main deployment.yaml.erb like so:
-
-```yaml
-<%= partial "cron", name: "cleanup",   schedule: "0 0 * * *", args: %w(cleanup),    cpu: "100m", memory: "100Mi" %>
-<%= partial "cron", name: "send-mail", schedule: "0 0 * * *", args: %w(send-mails), cpu: "200m", memory: "256Mi" %>
-```
-
-Inside a partial, parameters can be accessed as normal variables, or via a hash called `locals`. Thus, the `cron` template could like this:
-
-```yaml
----
-apiVersion: batch/v1beta1
-kind: CronJob
-metadata:
-  name: cron-<%= name %>
-spec:
-  schedule: <%= schedule %>
-  successfulJobsHistoryLimit: 3
-  failedJobsHistoryLimit: 3
-  concurrencyPolicy: Forbid
-  jobTemplate:
-    spec:
-      template:
-        spec:
-          containers:
-          - name: cron-<%= name %>
-            image: ...
-            args: <%= args %>
-            resources:
-              requests:
-                cpu: "<%= cpu %>"
-                memory: <%= memory %>
-          restartPolicy: OnFailure
-```
-
-Both `.yaml.erb` and `.yml.erb` file extensions are supported. Templates must refer to the bare filename (e.g. use `partial: 'cron'` to reference `cron.yaml.erb`).
-
-##### Limitations when using partials
-
-Partials can be included almost everywhere in ERB templates. Note: when using a partial to insert additional key-value pairs to a map you must use [YAML merge keys](http://yaml.org/type/merge.html). For example, given a partial `p` defining two fields 'a' and 'b',
-
-```yaml
-a: 1
-b: 2
-```
-
-you cannot do this:
-
-```yaml
-x: yz
-<%= partial 'p' %>
-```
-
-hoping to get
-
-```yaml
-x: yz
-a: 1
-b: 2
-```
-
-but you can do:
-
-```yaml
-<<: <%= partial 'p' %>
-x: yz
-```
-
-This is a limitation of the current implementation.
-
+All templates must be YAML formatted.
```

Comment:
Maybe it'd be worth it to mention "You may render ERB with `krane render` and then pipe that result to krane deploy --stdin`".